Boards/boards.xml

```diff
@@ -37,237 +37,242 @@
-    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e0eeb869eb1c6203a513aaefdf7fe27b1a4494fd/Boards/TargetBoardRX65N_V1.00.bcdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX65N_V1.12.bdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX65N_V1.00.bcdf"/>
   </board>
   <board id="TargetBoardRX660" version="1.01" description="Target Board for RX660 Board Description File" keys="RX">
-    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e0eeb869eb1c6203a513aaefdf7fe27b1a4494fd/Boards/TargetBoardRX660_V1.01.bdf"/>
-    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e0eeb869eb1c6203a513aaefdf7fe27b1a4494fd/Boards/TargetBoardRX660_V1.00.bcdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX660_V1.01.bdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX660_V1.00.bcdf"/>
   </board>
   <board id="TargetBoardRX66N" version="1.02" description="Target Board for RX66N Board Description File" keys="RX">
-    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e0eeb869eb1c6203a513aaefdf7fe27b1a4494fd/Boards/TargetBoardRX66N_V1.02.bdf"/>
-    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e0eeb869eb1c6203a513aaefdf7fe27b1a4494fd/Boards/TargetBoardRX66N_V1.01.bcdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX66N_V1.02.bdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX66N_V1.01.bcdf"/>
   </board>
   <board id="TargetBoardRX671" version="1.02" description="Target Board for RX671 Board Description File" keys="RX">
-    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e0eeb869eb1c6203a513aaefdf7fe27b1a4494fd/Boards/TargetBoardRX671_V1.02.bdf"/>
-    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e0eeb869eb1c6203a513aaefdf7fe27b1a4494fd/Boards/TargetBoardRX671_V1.01.bcdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX671_V1.02.bdf"/>
+    <download uri="https://raw.githubusercontent.com/renesas/smart-configurator-data/e6e4c24f943bb37708b7fe68cb0c66bc4ddcef1f/Boards/TargetBoardRX671_V1.01.bcdf"/>
   </board>
-</boards>
```

Comment:
Q: this line is removed?

Reply:
Sorry for missing this, made mistake when copy paste, added.